Apps/WebClient/src/Server/Controllers/UserProfileController.cs

```diff
@@ -46,7 +46,7 @@ public class UserProfileController : ControllerBase
         private readonly IUserProfileService userProfileService;
         private readonly IHttpContextAccessor httpContextAccessor;
         private readonly IUserEmailService userEmailService;
-        private readonly IUserSMSService userSMSService;
+        private readonly IUserSmsService userSMSService;
```

Comment:
Not sure forcing the variables names to be camel case for abbreviations is good

Reply:
We need to discuss this if its a concern

Reply:
The result of the discussion was that these naming changes will be rolled back and the rule should be disabled so that we can use abbreviations as we please.

Reply:
I reverted the Pascal naming changes and disabled the rule